website/src/get-started/wordcount-example.md

```diff
@@ -1267,7 +1267,12 @@ each element in the `PCollection`.
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+import apache_beam as beam
+from apache_beam.transforms import window
```

Comment:
Why do you need this import?

Reply:
I thought to show from where the are imported. But I guess developer can easily figure it out from programming guide.